src/assemblyline.c

```diff
@@ -28,6 +28,19 @@
 #include <sys/mman.h>
 #include <sys/stat.h>
 
```

Comment:
please add a little description, that this is the lookup table generation,

Reply:
Sure, I will get that done.